csaf_2.0/register/security-dot-txt/csaf.txt

```diff
@@ -0,0 +1,6 @@
+Field Name: Canonical
```

Comment:
I didn't check the other ones yet - but shouldn't that be `CSAF` instead of `Canonical` as the field in the security.txt should be called `CSAF`?